client/gutenberg/extensions/map-block/edit.js

```diff
@@ -0,0 +1,146 @@
+/** @format */
+
+/**
+ * Wordpress dependencies
+ */
+
+import { __ } from '@wordpress/i18n';
+
+import { Component, createRef, Fragment } from '@wordpress/element';
+
+import { IconButton, PanelBody, Toolbar } from '@wordpress/components';
+
+import {
+	InspectorControls,
+	BlockControls,
+	BlockAlignmentToolbar,
+	PanelColorSettings,
+} from '@wordpress/editor';
+
+/**
+ * External dependencies
+ */
+
+import { clone } from 'lodash';
```

Comment:
Just to have one fewer dependency, what are your thoughts on instead doing something like `let clonedArray = originalArray.slice(0);` which I believe should have the same effect?

https://davidwalsh.name/javascript-clone-array

Reply:
Good call. Fixed in https://github.com/Automattic/wp-calypso/pull/27836/commits/3bfbe0c922c1ffd2c183aa2bc85a2922a71a1154